src/main/java/entities/Listing.java

```diff
@@ -43,22 +62,10 @@ public float getPrice() {
         return price;
     }
 
-    public User getSeller() {
```

Comment:
Why are are removing these getters and setters? Is it violating clean code? Will need to discuss during meeting

Reply:
we refactored the Listing Class to only store a sellerUsername and a sellerRating instead of a whole User object. Thus there is no seller attribute anymore. this is due to circular references in the database that were difficult to resolve. if you still need the User, you can use the getUserWithUsername method